test/test_model_regression.py

```diff
@@ -70,18 +70,18 @@ def md5(fname):
 #       largest relative errors seem to be post-merger
 #       only seems to affect models that use gpr fits and/or gsl calls
 rtol                   = 1.e-11
-rtol_NRHybSur3dq8      = 2.e-5
+rtol_NRHybSur3dq8      = 2.e-5  # used for GPR-fit models "NRHybSur3dq8", "NRHybSur2dq15"
 rtol_NRHybSur3dq8Tidal = 3.e-4
 rtol_SpEC_q1_10_NoSpin_linear_alt = 3.e-8 # needed for (8,7) mode to pass. Other modes pass with 1e-11 tolerance
 
 # TODO: new and old surrogate interfaces should be similar enough to avoid
 #       model-specific cases like below
 
 # Old surrogate interface
-surrogate_old_interface = ["SpEC_q1_10_NoSpin","EOBNRv2_tutorial","EOBNRv2","SpEC_q1_10_NoSpin_linear","EMRISur1dq1e4"]
+surrogate_old_interface = ["SpEC_q1_10_NoSpin","EOBNRv2_tutorial","EOBNRv2","SpEC_q1_10_NoSpin_linear","EMRISur1dq1e4","BHPTNRSur1dq1e4"]
 
 # news loader class
-surrogate_loader_interface = ["NRHybSur3dq8","NRHybSur3dq8Tidal","NRSur7dq4","NRHybSur2d15"]
+surrogate_loader_interface = ["NRHybSur3dq8","NRHybSur3dq8Tidal","NRSur7dq4","NRHybSur2dq15"]
```

Comment:
I'm confused; why this even work before?

Reply:
It didn't! the q15 test was disabled until this PR